packages/angular_devkit/schematics_cli/BUILD.bazel

```diff
@@ -20,19 +21,46 @@ ts_library(
             "bin/**/*_benchmark.ts",
         ],
     ),
-    module_name = "@angular-devkit/schematics-cli",
-    module_root = "bin",
     # The attribute below is needed in g3 to turn off strict typechecking
     # strict_checks = False,
+    data = glob(["**/*.json"]),
+    module_name = "@angular-devkit/schematics-cli",
+    module_root = "bin/index.d.ts",
```

Comment:
Can `module_root = "bin/schematics.d.ts",` be used instead?  no need for the extra and otherwise unused file then.

Reply:
I had tried that before, but it doesn't seem to work. Deleting the `bin/index.ts` file and adding `bin/schematics.d.ts` here instead I saw this error:
```
ERROR: /home/filipesilva/work/cli/packages/angular_devkit/schematics_cli/BUILD.bazel:14:1: Compiling TypeScript (devmode) //packages/angular_devkit/schematics_cli:schematics_cli failed (Exit 1)
Compilation failed Error: ERROR: /home/filipesilva/.cache/bazel/_bazel_filipesilva/e327cb0949caea626cba080635966a9f/execroot/angular_cli/packages/angular_devkit/schematics_cli/bin/schematics.ts contains a module name declaration @angular-devkit/schematics-cli/bin/schematics which would be overwritten with @angular-devkit/schematics-cli by Bazel's TypeScript compiler.
```

To be honest I don't know why. I kinda chalked it up to `rules_typescript` quirkiness and left it at that because it didn't seem very important at the time.

I also tried `bin/schematics` instead because I saw it in https://github.com/angular/angular/blob/fc3e5cb6d3ef61aaa13bd9788b9b1c7b97af94fe/packages/core/test/bundling/util/BUILD.bazel#L12, but same error.

Then I tried doing `bazel clean` and retrying the above, and the error was gone. So yeah it works but I guess Bazel was being non-deterministic. Thanks for prodding me to look at it again.